api/index.js

```diff
@@ -93,7 +93,12 @@ export default async (req, res) => {
       }),
     );
   } catch (err) {
-    res.setHeader("Cache-Control", `no-cache, no-store, must-revalidate`); // Don't cache error responses.
+    res.setHeader(
+      "Cache-Control",
+      `max-age=${CONSTANTS.ERROR_CACHE_SECONDS / 2}, s-maxage=${
+        CONSTANTS.ERROR_CACHE_SECONDS
+      }, stale-while-revalidate=${CONSTANTS.ONE_DAY}`,
```

Comment:
Whats happening here? 

`CONSTANTS.ERROR_CACHE_SECONDS / 2`  in max-age
`CONSTANTS.ERROR_CACHE_SECONDS` in s-maxage
`CONSTANTS.ONE_DAY` in revalidate?

Reply:
wait this code is on master lol. Have no idea why is it like this 😆

Reply:
Okay this was the change https://github.com/anuraghazra/github-readme-stats/pull/2124

Reply:
Yea, this was when I would sometimes merge PRs without your review since they had been open for a long time. Sorry for the confusion 😅. Feel free to change it to any caching behavior you like. The reasoning for the current caching code can be found at https://github.com/anuraghazra/github-readme-stats/pull/2124#issuecomment-1282864119.

Reply:
@anuraghazra, we can also remove these lines and keep the same behavior as for the case when no error is thrown. Both are fine with me.

Reply:
Let's keep the same behaviour as old one. Not really confident with how this header will behave on prod.